docs/source/supported_layers.md

```diff
@@ -0,0 +1,183 @@
+ ### Pooling Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  | Fabrik Caffe  | Fabrik Keras    | Fabrik Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: | :-----------: | :--------------: | :-----------------: | 
+| MaxPooling1D              | √             | √            | √           | √             | √               | ×                    |
+| MaxPooling2D              | √             | √            | √           | √             | √               | √                    |
+| MaxPooling3D              | √             | √            | √           | √             | √               | √                    |
+| AveragePooling1D          | √             | √            | √           | ×             | √               | ×                    |
+| AveragePooling2D          | √             | √            | √           | ×             | √               | ×                    |
+| AveragePooling3D          | √             | √            | √           | ×             | √               | ×                    |
+| GlobalMaxPooling1D        | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalAveragePooling1D    | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalMaxPooling2D        | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalAveragePooling2D    | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalMaxPooling3D        | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalAveragePooling3D    | ×             | √            | √           | ×             | ×               | ×                    |
+| Stochastic Pooling        | √             | ×            | ×           | √             | ×               | ×                    |
+ 
+ ### Data Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  | Fabrik Caffe  | Fabrik Keras    | Fabrik Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: | :-----------: | :--------------: | :-----------------: | 
+| Image Data                | √             | √            | √           | √             | √               | √                    |
+| Data                      | √             | √            | √           | √             | √               | √                    |
+| HDF5 Data                 | √             | √            | √           | √             | √               | √                    |
+| HDF5 Output Data          | √             | √            | √           | √             | √               | √                    |
+| Input                     | √             | √            | √           | √             | √               | √                    |
```

Comment:
If i remember correctly, for Keras and Tensorflow (in Fabrik) only `Input` is available. All other data layers are not available.